comparative_benchmark/pt_inductor/benchmark_all.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
nit: Add example usage and environment variables.

Reply:
Done.